src/test/java/kitchenpos/application/OrderTableServiceTest.kt

```diff
@@ -0,0 +1,273 @@
+            }
+        }
+    }
+
+    @Nested
+    inner class `테이블 앉기 테스트` {
+        @DisplayName("존재하지 않는 id 라면, NoSuchElementException 예외 처리한다.")
+        @Test
+        fun test1() {
+            // given
+            val orderTableId = UUID.randomUUID()
+
+            every { orderTableRepository.findById(any()) } returns Optional.empty()
+
+            // when & then
+            shouldThrowExactly<NoSuchElementException> {
+                orderTableService.sit(orderTableId)
+            }
+        }
+
+        @DisplayName("정상적인 요청이라면, 테이블 점유 상태가 된다.")
+        @Test
+        fun test2() {
+            // given
+            val orderTableId = UUID.randomUUID()
+            val 앉으려는_테이블 = createOrderTableRequest(
+                isOccupied = false,
+            )
+
+            val 찾아온_테이블 = createOrderTable(
+                id = orderTableId,
+                isOccupied = 앉으려는_테이블.isOccupied
+            )
+
+            every { orderTableRepository.findById(any()) } returns Optional.of(찾아온_테이블)
+
+            // when
+            val result = orderTableService.sit(orderTableId)
+
+            // then
+            result.isOccupied shouldBe true
+        }
+    }
+
+    @Nested
+    inner class `테이블 치우기 테스트` {
+        @DisplayName("존재하지 않는 테이블 아이디이면, NoSuchElementException 예외 처리를 한다.")
+        @Test
+        fun test1() {
+            // given
+            val orderTableId = UUID.randomUUID()
+
+            every { orderTableRepository.findById(any()) } returns Optional.empty()
+
+            // when & then
+            shouldThrowExactly<NoSuchElementException> {
+                orderTableService.clear(orderTableId)
+            }
+        }
+
+        @DisplayName("주문 상태가 완료인 주문을 제외하고, 해당 테이블이 속해있는 주문이 있다면, IllegalStateException 예외 처리를 한다.")
+        @Test
+        fun test2() {
+            // given
+            val orderTableId = UUID.randomUUID()
+
+            val 찾아온_테이블 = createOrderTable(id = orderTableId)
+
+            every { orderTableRepository.findById(any()) } returns Optional.of(찾아온_테이블)
+            every { orderRepository.existsByOrderTableAndStatusNot(any(), any()) } returns true
+
+            // when & then
+            shouldThrowExactly<IllegalStateException> {
+                orderTableService.clear(orderTableId)
+            }
+        }
+
+        @DisplayName("정상 요청이라면, 해당 테이블을 치운다.")
+        @Test
+        fun test3() {
+            // given
+            val orderTableId = UUID.randomUUID()
+
+            val 찾아온_테이블 = createOrderTable(
+                id = orderTableId
+            )
+
+            every { orderTableRepository.findById(any()) } returns Optional.of(찾아온_테이블)
+            every { orderRepository.existsByOrderTableAndStatusNot(any(), any()) } returns false
+
+            // when
+            val result = orderTableService.clear(orderTableId)
+
+            // then
+            result.numberOfGuests shouldBe 0
+            result.isOccupied shouldBe false
+        }
```

Comment:
사소한 부분이긴 한데 orderTabledId를 별도로 선언하지 않고 아래처럼 해도 되지 않을까요?
```suggestion
        fun test3() {
            // given       
            val 찾아온_테이블 = createOrderTable()
     
            every { orderTableRepository.findById(any()) } returns Optional.of(찾아온_테이블)
            every { orderRepository.existsByOrderTableAndStatusNot(any(), any()) } returns false

            // when
            val result = orderTableService.clear(찾아온_테이블.id)

            // then
            result.numberOfGuests shouldBe 0
            result.isOccupied shouldBe false
        }
```